config/index.js

```diff
@@ -271,6 +271,25 @@ var conf = convict({
       default: ''
     }
   },
+  verificationReminder: {
+    enabled: {
+      doc: 'Is verification reminder enabled',
+      format: Boolean,
+      env: 'VERIFICATION_REMINDER_ENABLED',
+      default: false
+    },
+    queueUrl: {
+      doc: 'The verification reminder queue URL to use (should include https://sqs.<region>.amazonaws.com/<account-id>/<queue-name>)',
+      format: String,
+      env: 'VERIFICATION_REMINDER_QUEUE_URL',
```

Comment:
> However I wanted to test real SQS locally, would the aws-sdk module look for my AWS creds via process.env?

I'm not sure thats such a great idea or necessary, unless you're only doing it for now while you're getting started. It complicates the permissions to SQS. I think we should keep it internal network only.

credentials: via `credentials` file http://aws.amazon.com/sdk-for-node-js/
policy: http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/UsingIAM.html